Source/MQTTnet/Server/Internal/MqttClientConnectionStatistics.cs

```diff
@@ -32,7 +32,13 @@ public MqttClientConnectionStatistics()
             _lastNonKeepAlivePacketReceivedTimestamp = _connectedTimestamp;
         }
 
-        public DateTime LastPacketReceivedTimestamp => _lastPacketReceivedTimestamp;
+        // This class is tracking all values from the Client's perspective regarding the internal "received"/"sent" timestamps.
+        // "Sent" means, a package has been received from the client.
+        // "Received" means, a package has been sent by the server.
+        // The public property below makes it less ambiguous.
+
+        public DateTime LastPacketReceivedFromClientTimestamp => _lastPacketSentTimestamp;
```

Comment:
Please do not rename the property for now. This will cause breaking changes. The comment can be wrapped into a summary tag so that every user can read this explanation.

Reply:
True. I have modified the code to expose the additional required `LastPacketSentTimestamp` for use in the keep alive monitor and added "Summary" comments to both, the "Sent" and "Received" properties. Please have a look.